scripts/install_gotool.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# install_gotool.sh is a utility script used by the Make process
+# for installing a pinned dependency from the tools/tools.go file.
+# It accepts two arguments:
+#
+# Arg1: the name of the tool, i.e. `goimports`
+# Arg2: the fully qualified GitHub URL, i.e., `golang.org/x/tools/cmd/goimports`
+#
+# This script is not meant to be utilized independently, the installation
+# of Go tools should be performed via the make process
+
+location=$(command -v ${1})
```

Comment:
We should really be looking at the location where our process places the tools. In most cases that will be $GOBIN but it could just as easily be build/tools or similar.

Basically, relying on the path means we’re not using the tool we think we should be using.

Reply:
@sykesm  I assume you are suggesting we select the spot here (and I suppose we'll need to add that to our PATH here as well)?  Probably best to use our own folder in case people have other versions of the tool they prefer already in $GOBIN?